File: WorkIt/app/src/main/java/prot3ct/workit/view_models/IsUserAssignableToTaskViewModel.java
package prot3ct.workit.view_models;

public class IsUserAssignableToTaskViewModel {
    private int pendingRequestId;
    private String isUserAssignableToTaskMessage;

    public String getIsUserAssignableToTaskMessage() {
        return isUserAssignableToTaskMessage;
    }

    public void setIsUserAssignableToTaskMessage(String isUserAssignableToTaskMessage) {
        this.isUserAssignableToTaskMessage = isUserAssignableToTaskMessage;
    }

    public int getPendingRequestId() {
        return pendingRequestId;
    }

    public void setPendingRequestId(int pendingRequestId) {
        this.pendingRequestId = pendingRequestId;
    }
}

File: WorkIt-Server/WorkIt-Server/Models/ViewModels/IsUserAssignableToTaskViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WorkIt_Server.Models.ViewModels
{
    public class IsUserAssignableToTaskViewModel
    {
        public int PendingRequestId { get; set; }
        public string IsUserAssignableToTaskMessage { get; set; }
    }
}